nordicsemi/dfu/dfu_transport_serial.py

```diff
@@ -180,6 +180,7 @@ def __init__(self,
 
 
     def open(self):
+        time.sleep(8)
```

Comment:
There is already a sleep in [dfu.js](https://github.com/andvib/pc-nrfutil/blob/1a1743063a66896ae736c23cd7e27e35f88185bc/nordicsemi/dfu/dfu.py#L82). Wouldn't it be sufficient to modify that value?

Reply:
That would be sufficient, yes. However, this delay is only needed for the serial transport, and adding it to dfu.py would introduce the delay for BLE transport as well. So I guess it's a question whether or not to slow down the BLE transport as well?

Reply:
As you say @andvib, increasing the delay in dfu.py will slow down other transports, so we should avoid that. Adding hard-coded sleep values is, in general, not a good solution.

If a delay is required before connecting, and this delay depends on the DFU type/payload and target device, then perhaps we need to make this configurable as an optional parameter? 8 seconds may work for your use case, but I suspect there will be cases where this is too much or too little.

Reply:
@andvib Will #129 cover your use case?

Reply:
Yes, #129 will work great, thanks! I'll close this PR.